lib/ansible/playbook/role_include.py

```diff
@@ -113,17 +113,17 @@ def load(data, block=None, role=None, task_include=None, variable_manager=None,
             raise AnsibleParserError("'name' is a required field for include_role.")
 
         # validate bad args, otherwise we silently ignore
-        bad_opts = my_arg_names.difference(IncludeRole.VALID_ARGS)
+        bad_opts = my_arg_names.difference(set(IncludeRole.VALID_ARGS))
```

Comment:
Note: you don't have to use set explicitly here.  The set methods can work with an iterable (which tuple is)